src/app/courses/index/index.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="index-component">
   <section class="item-component" *ngFor="let item of listCourses">
     <ul>
-      <app-item [data]="item"></app-item>
+      <app-item (childEvent)='deleteCourse($event)' [data]='item'></app-item>
```

Comment:
childEvent should be renamed to delete or smth related to it in this case, you are using this output to delete an output, "childEvent" doesn't explain anything about this output